egs/voxceleb/v1/local/make_voxceleb1.pl

```diff
@@ -1,130 +1,127 @@
 #!/usr/bin/perl
 #
-# Copyright 2018  Ewald Enzinger
-#           2018  David Snyder
```

Comment:
You can put your own copyright in the new version of the file (make_voxceleb_v2.pl), if you want.